rust/ql/test/library-tests/frameworks/rusqlite/main.rs

```diff
@@ -24,27 +24,27 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         )",
         (),
     )?;
-    
+
     let query = format!("INSERT INTO person (name, age) VALUES ('{}', '{}')", name, age);
 
     connection.execute(&query, ())?;    // $ sql-sink
 
     let person = connection.query_row(&query, (), |row| {    // $ sql-sink
         Ok(Person {
```

Comment:
Just insert the following and record the failure cause as a type inferencer test.
```rust
        let row: &rusqlite::Row<'_> = row;
``